### DISCUSSION TITLE
I want to make two types of release file one is major release and second is minor release.

### DISCUSSION THREAD

**Discussion by ansulagrawal** [semantic-release/semantic-release#2819] [Release workflow]
I have a release yml file : https://github.com/ansulagrawal/react-big-schedule/blob/master/.github/workflows/publish.yml
that publish on npm as well as it make a tag, Its makes major release i.e 1.0.0 to 2.0.0.

I want to make another yml file for release but with smaller release i.e. 2.0.0 to 2.0.1 or 2.0.9 to 2.1.0.
how should I do it can someone help me out for same.

I tried all the way to do it but didn't find out solution.

Please help me out.

Thanks in advance.

**Comment by travi** [comment]
what you describe isnt really how semantic-release works. publishing major, minor, and patch releases are all handled based on what is communicated in the commits based on the commit convention. is there a reason that you are wanting to split the process across multiple workflows specifically with semantic-release?

**Reply by ansulagrawal** [reply]
Yes I want to do major release every month end and minor release on every week. Thats why I want to shift 2.0 to 3.0 on major release and 2.0.0 to 2.0.1 on minor release.

**Reply by ansulagrawal** [reply]
Or can you suggest how should I commit the code so it make minor release?

**Reply by travi** [reply]
> Yes I want to do major release every month end and minor release on every week.

according to semver, major versions are to communicate breaking changes, not a calendar based cadence. what you are describing aligns better to calver than semver. semantic-release is specifically intended to simplify the use of semver, so semantic-release may not be a fit for your goals.

**Reply by ansulagrawal** [reply]
Can you share me the link please to check it out.

**Reply by travi** [reply]
support provided by maintainers of this project is limited to use of the project itself and even that stretches us thin. for research beyond the scope of this project, we have to ask you to put that effort in yourself.

**Reply by ansulagrawal** [reply]
I saw antd library it uses Semantic Versioning 2.0.0 that is same way I want it.
